Packs/CortexVulnerabilityManagement/Playbooks/Cortex_VM_-_ServiceNow_CMDB_README.md

```diff
@@ -6,35 +6,38 @@ This playbook uses the following sub-playbooks, integrations, and scripts.
 
 ### Sub-playbooks
 
-ServiceNow CMDB Search
+* ServiceNow CMDB Search
 
 ### Integrations
 
-ServiceNow v2
+* ServiceNow v2
 
 ### Scripts
 
-GridFieldSetup
+* ContextSetup
 
 ### Commands
 
-servicenow-query-users
+* servicenow-query-users
 
 ## Playbook Inputs
 
 ---
 
 | **Name** | **Description** | **Default Value** | **Required** |
 | --- | --- | --- | --- |
-| IPv4 | IPv4 address. |  | Optional |
-| IPv6 | IPv6 address. |  | Optional |
-| HostName | Hostname. |  | Optional |
-| FQDN | Fully Qualified Domain Name. |  | Optional |
+| IPv4 | IPv4 address |  | Optional |
+| IPv6 | IPv6 address |  | Optional |
+| HostName | Hostname |  | Optional |
+| FQDN | Fully Qualified Domain Name |  | Optional |
```

Comment:
Descriptions should always have a period at the end of the description.  Not sure why they were removed here.

Reply:
seems like issue with SDK